website/docs/reference/deploy/environment-import-export.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Environment Import & Export
+---
+
+:::info Availability
+
+The environment import and export first appeared in Unleash 4.22.0.
+
+:::
+
+The environment export/import functionality in Unleash simplifies the process of moving feature flags between multiple instances. 
+It also makes it easier to move feature flags within a single instance between environments.
+When exporting you always select the toggles and environment to export from. When importing you always select a project and an environment to import to.
+
+## Import & Export items
+
+The following items will be exported and imported:
+* feature toggles
+* activation strategies
+* variants
+* enabled/disabled statuses
+* context fields
+* feature tags
+* feature tag types
+* segments
+
+## Export
+
+Export is search-driven based on the results you see in the project toggle search or a global toggle search.
```

Comment:
As a reader: what does search-driven mean? Is it something like this:

```suggestion
You can export features either from a project search or the global feature search. Use the search functionality to narrow the results to the list of features you want to export and use the export button to select environment. All features included in your search results will be included in the export.
```

But still, this feels very UI-centric and may be more at home in a how-to guide. How does the API work?

Reply:
This feature is UI centric as UI import is our main use case. I took the wording from our import/export article.

Reply:
At the API level you have a list of feature IDs to export and the env name